libs/libc/tls/Make.defs

```diff
@@ -21,8 +21,7 @@
 CSRCS += task_getinfo.c tls_getinfo.c
 
 ifneq ($(CONFIG_TLS_NELEM),0)
-CSRCS += tls_alloc.c tls_free.c
-CSRCS += tls_setvalue.c tls_getvalue.c tls_destruct.c
+CSRCS += tls_destruct.c
```

Comment:
Maybe can move `tls_destruct.c` and name it `pthread_keydestruct.c`?

Reply:
But functions inside tls_destruct.c isn't specified by POSIX.